src/base/emu-i386/simx86/interp.c

```diff
@@ -511,6 +511,15 @@ static unsigned int _Interp86(unsigned int PC, int basemode)
 		OVERR_SS = Ofs_XSS;
 		TheCPU.mode = mode = basemode;
 
+		if ((CONFIG_CPUSIM & CPUSIM_INSTREMU) && (!(CEmuStat & CeS_INHI))) {
+			if (debug_level('e')>1)
+				dbug_printf("CPUSIM_INSTREMU, count=%d\n",
+					    TheCPU.inst_emu_count);
+			if (TheCPU.inst_emu_count-- == 0 || signal_pending()) {
```

Comment:
As per 9e7bbd9c6 you need to use
`TheCPU.sigalrm_pending` instead.

Reply:
done